src/azure-cli/azure/cli/command_modules/botservice/channel_operations.py

```diff
@@ -19,6 +23,25 @@ def create_channel(client, channel, channel_name, resource_group_name, resource_
     )
 
 
+def update_channel(client, channel, channel_name, resource_group_name, resource_name):
```

Comment:
One more question here is that channel_name is required for you function. But where do you use channel_name here?

Reply:
I've remedied this by addressing @jsntcy's [feedback](https://github.com/Azure/azure-cli/pull/11318#discussion_r352514008).